packages/terra-alert/src/Alert.module.scss

```diff
@@ -33,6 +33,16 @@
     margin-top: var(--terra-alert-icon-margin-top);
   }
 
+  .content-container {
+    align-items: flex-start;
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    line-height: 1.2;
```

Comment:
Should we be specifying a line height or let it be inherited.  Also, shouldn't the line height have units if it is required?

Reply:
This styling was there before the change, I just changed an HTML layout a bit hence had to move some styling around per new class names, so it looks like a new code here, but it has been there a long time and it's out of the scope of this JIRA to change or keep it.

Reply:
Here is the source of this styling: https://github.com/cerner/terra-core/blob/5f0141ba64d5ae282d5321629e7fdb0385884b18/packages/terra-alert/src/Alert.module.scss#L41

Reply:
ok.  It just looked new based on the changes.